prog.go

```diff
@@ -41,6 +42,9 @@ type ProgramOptions struct {
 	// Controls the output buffer size for the verifier. Defaults to
 	// DefaultVerifierLogSize.
 	LogSize int
+	// The ELF file containing the BTF to relocate against for CO-RE. May be
+	// nil, which implies relocating against the kernel.
```

Comment:
Is there an alternative description we could give instead of `relocating against the kernel`. Perhaps something that explicitly indicates that setting`TargetBTF` overrides the default kimg scanning routine, or that it might apply to containerized binaries? Taking some parts of the commit message would improve this. :+1:

Reply:
I reworded it a bit, hopefully it's better now.

Reply:
Perfect! :ok_hand: